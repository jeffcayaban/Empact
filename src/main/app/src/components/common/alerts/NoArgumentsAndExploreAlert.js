import React from 'react';
import { Link } from "react-router-dom";
import { Alert } from "react-bootstrap";

/**
 * Is the alert for when there are no arguments and invites the user to explore other petitions.
 */

class NoArgumentsAndExploreAlert extends React.PureComponent {
    render() {
        return (
            <Alert bsStyle="warning">
                There are no arguments here! Click <Link to={'/petitions'}>here</Link> to explore some petitions!
            </Alert>
        );
    }
}

export default NoArgumentsAndExploreAlert;