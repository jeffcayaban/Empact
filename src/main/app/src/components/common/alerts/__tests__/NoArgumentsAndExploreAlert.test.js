import React from 'react';
import toJson from "enzyme-to-json";
import { shallow } from "enzyme";
import NoArgumentsAndExploreAlert from "../NoArgumentsAndExploreAlert";

describe('NoArgCreationOnClosedPetitionAlert', () => {

    it('should render successfully', () => {
        const component = shallow(<NoArgumentsAndExploreAlert />);
        expect(toJson(component)).toMatchSnapshot();
    });
});
