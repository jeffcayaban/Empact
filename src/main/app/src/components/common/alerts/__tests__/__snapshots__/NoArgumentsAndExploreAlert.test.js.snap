// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoArgCreationOnClosedPetitionAlert should render successfully 1`] = `
<Alert
  bsClass="alert"
  bsStyle="warning"
  closeLabel="Close alert"
>
  There are no arguments here! Click 
  <Link
    replace={false}
    to="/petitions"
  >
    here
  </Link>
   to explore some petitions!
</Alert>
`;
